lib/nylas.rb

```diff
@@ -128,12 +128,12 @@ def url_for_path(path)
 
     def url_for_authentication(redirect_uri, login_hint = '', options = {})
       params = {
-        :client_id => @app_id,
-        :trial => options.fetch(:trial, false),
+        :client_id     => @app_id,
+        :trial         => options.fetch(:trial, false),
         :response_type => 'code',
-        :scope => 'email',
-        :login_hint => login_hint,
-        :redirect_uri => redirect_uri,
+        :scope         => options.fetch(:scope, 'email'),
```

Comment:
This defaults to the generic `email` scope in the case that more specific email scopes (like `'email.modify` and `email.send`) are not provided? If no `:scope` option was passed in, and `email` is used, is that equivalent to what our code currently does?

Reply:
This code functions identically to the current version of the gem if no options are passed in. And yes, this code would keep our app the same if no options are passed in.